tests/drink/src/psp22_tests.rs

```diff
@@ -0,0 +1,77 @@
+use anyhow::Result;
+use assert2::assert;
+use drink::{runtime::MinimalRuntime, session::Session};
+use ink_primitives::AccountId;
+use ink_wrapper_types::{util::ToAccountId, Connection};
+use psp22_contract::{Instance, PSP22 as _};
+
+use crate::*;
+
+pub fn balance_of(
+    session: &mut Session<MinimalRuntime>,
+    psp22: Instance,
+    account: AccountId,
+) -> u128 {
+    session
+        .query(psp22.balance_of(account))
+        .unwrap()
+        .result
+        .unwrap()
+}
+
+pub fn setup(session: &mut Session<MinimalRuntime>, caller: drink::AccountId32) -> Instance {
+    let _code_hash = session.upload_code(psp22_contract::upload()).unwrap();
+
+    let _ = session.set_actor(caller);
+
+    session
+        .instantiate(Instance::new(1000))
```

Comment:
why don't you use `session.deploy` here?

Reply:
IIRC I was mimicking the aleph_client e2e tests. I can use `deploy` in one of the tests but I think showcasing all of the functionality (and testing it) is desired.

Reply:
sounds fair, resolving

Reply:
Actualy... There's no `deploy` on `trait Connection` from `ink-wrapper-types`, that's why it's not available here on `Session<_>` object.